src/docs/upgrade.dox

```diff
@@ -85,5 +85,8 @@ To downgrade to a previous release of WiredTiger:
 -# Shut down the new version of the application.
 <br><br>
 -# Start the old version of the application.
+<br><br>
+-# The database created with version 10.0 cannot be downgraded to older
+   versions.
```

Comment:
The documentation has a [page on building with POSIX](http://source.wiredtiger.com/3.2.1/build-posix.html) that lists some of the --enable/--disable options, maybe it needs updating?

It may need updating in general if it doesn't match reality.

We may no longer be updating the documentation at all, too, so ignore this comment if that's the case.

Reply:
This [page](http://source.wiredtiger.com/3.2.1/build-posix.html) has a configuration list that needs to be enabled explicitly, but the `standalone-build` option will be enabled by default in WiredTiger and the disable option which has been added is mostly for MongoDB. Please let me know if you want me to update it.